liveRecorder/serviceWrappers/linux/liveRecorder

```diff
@@ -29,8 +29,8 @@ SCRIPT_DIR=$(dirname $(readlink -f $0))
 APPDIR=${SCRIPT_DIR}/../../
 NAME=${APP_NAME}
 APPLICATION_PATH=${APPDIR}/main.py
-LOG_DIR="/var/log/${APP_NAME}"
-PID_DIR="/var/run"
+LOG_DIR="@LOG_DIR@/${APP_NAME}"
```

Comment:
I'm fine with saving this file as .template and the original file in the saas-config, so the rpm installation will create from template.
I'm also fine with loading the default values for environment variables from /etc/default, if a file exists.